utils/spaceship.scriptLoader/test.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <button id="btn" onclick="myLoader()">submit</button>
+    <script>
+        function scriptLoader(src) {
```

Comment:
this scriptLoader function is repeated, even though you add util code in `utils/spaceship.scriptLoader/index.js`. I think it is better if you load function in `utils/spaceship.scriptLoader/index.js`.

So, you can reduce useless repeated code.

Reply:
Oh, Thanks for your review.
Um... I think it's better to remove `utils/spaceship.scriptLoader/test.html` file.
Because user don't need to download it in bundles to `test.html`
Instead, I'll write a good example on the document📝

Reply:
yeap, it sounds good to me, thanks! 👍